libwaysip/src/lib.rs

```diff
@@ -1,625 +1,30 @@
-mod error;
+mod dispatch;
 mod render;
-pub use error::WaySipError;
 
-use std::os::unix::prelude::AsFd;
+pub mod error;
+pub mod state;
 
```

Comment:
```rust
pub use state::SelectionType;
```

Emm. can you add this line to make it can use libwaysip::SelectionType directly?

Reply:
Yep I was just about to push a commit for it. On eminute